simpleclient/src/main/java/io/prometheus/client/Collector.java

```diff
@@ -103,13 +117,14 @@ public int hashCode() {
         hash = 37 * hash + labelValues.hashCode();
         long d = Double.doubleToLongBits(value);
         hash = 37 * hash + (int)(d ^ (d >>> 32));
+        hash = 37 * hash + timestamp.hashCode();
```

Comment:
this needs to be wrapped in an `if (timestamp != null)` check

Reply:
Thank you, I have changed it, but still not pushed.